lms/static/sass/_pearsonx.scss

```diff
@@ -660,9 +662,6 @@ sure that the inquiry success banner still appears correctly.
       font-size: 2.5em;
       padding: 20px;
     }
-    .msg-content {
```

Comment:
It seems this doesn't change the style because this part will still come from the upstream theme: https://github.com/edx/edx-platform/blob/master/lms/static/sass/elements/_system-feedback.scss#L135
But it doesn't hurt to remove it either if we're not using it. And I already tested that it works with this removed.